app/models/decidim/census_connector/person.rb

```diff
@@ -31,8 +31,29 @@ def born_at
         @born_at ||= Date.parse(person_data.born_at)
       end
 
+      def age
+        @age ||= calculate_age
+      end
+
       private
 
+      def calculate_age
+        now = Time.zone.now.to_date
+
+        this_year = now.year
+        this_month = now.month
+        today = now.day
+
+        birth_year = born_at.year
+        birth_month = born_at.month
+        birth_day = born_at.day
+
+        month_of_birth_passed = this_month > birth_month
+        month_of_birth_current_but_day_passed = this_month == birth_month && today >= birth_day
+
+        this_year - birth_year - (month_of_birth_passed || month_of_birth_current_but_day_passed ? 0 : 1)
+      end
+
```

Comment:
https://medium.com/@craigsheen/calculating-age-in-rails-9bb661f11303 :smile:

Reply:
:smiley: So cool! I'll update it!

Reply:
Mmm, maybe leap years would break things... :thinking:

Reply:
I took the code from stackoverflow and this was the most voted answer, precisely because of leap years problems. https://stackoverflow.com/questions/819263/get-persons-age-in-ruby

Reply:
yes, your version is clear and safe, forget about this comments :see_no_evil:

Reply:
I assumed if they wrote a blog post about it, it would be correct :rofl: